web/src/components/OpeningHours.tsx

```diff
@@ -46,13 +61,19 @@ const OpeningHours = ({
   isCurrentlyOpen,
   openingHours,
   isTemporarilyClosed,
+  link,
 }: OpeningHoursProps): ReactElement | null => {
   const { t } = useTranslation('pois')
 
   const openingHoursTitle = (
     <TitleContainer>
       <span>{t('openingHours')}</span>
-      <OpeningLabel $isOpen={isCurrentlyOpen}>{t(getOpeningLabel(isTemporarilyClosed, isCurrentlyOpen))}</OpeningLabel>
+      <OpeningContainer>
+        <OpeningLabel $isOpen={isCurrentlyOpen}>
+          {t(getOpeningLabel(isTemporarilyClosed, isCurrentlyOpen))}
+        </OpeningLabel>
+        <StyledIcon src={NoteIcon} />
```

Comment:
:question: Does this really make sense here? It is not in the design/is always displayed without any context.